pytorch_transformers/tokenization_utils.py

```diff
@@ -666,35 +696,146 @@ def _convert_token_to_id(self, token):
     def encode(self, text, text_pair=None, add_special_tokens=False, **kwargs):
         """
         Converts a string in a sequence of ids (integer), using the tokenizer and vocabulary.
-        
+
         Same as doing ``self.convert_tokens_to_ids(self.tokenize(text))``.
 
         Args:
             text: The first sequence to be encoded.
             text_pair: Optional second sequence to be encoded.
             add_special_tokens: if set to ``True``, the sequences will be encoded with the special tokens relative
                 to their model.
-            **kwargs: passed to the `self.tokenize()` method
```

Comment:
kwargs is still passed to the `tokenize()` methode isn't it?

Reply:
You're right, it shouldn't have been removed.